backend/stackdriver.go

```diff
@@ -37,22 +42,24 @@ func NewStackDriverBackend(gcpProjectID string) (*StackDriverBackend, error) {
 	}
 
 	return &StackDriverBackend{
-		projectId: 		gcpProjectID,
-		client: 		c,
-		metricTypes:    make(map[string]string),
+		projectID:   gcpProjectID,
+		client:      c,
+		metricTypes: make(map[string]string),
 	}, nil
 }
 
+// Collect metrics
 func (sd *StackDriverBackend) Collect(r *collector.Result) error {
 	ctx := context.Background()
 	now := &timestamp.Timestamp{
 		Seconds: time.Now().Unix(),
 	}
+	orgName := dashReplacer.Replace(r.Org)
```

Comment:
We should probably use a better replacer here that replaces all illegal characters with underscores. The set of allowed characters as documented [here](https://cloud.google.com/monitoring/api/v3/metrics-details#label_names) is:

- You can use upper and lower-case letters, digits, and underscores (_) in the names.
- You can use periods (.) in the domain part of the names.
- You can use forward slashes (/) to separate path elements.
- You can start each path element with a letter or digit.
- The maximum length of a metric type name is 200 characters.

Which sounds like we should maybe do this:

```go
illegalCharacters := regexp.MustCompile(`[^A-Za-z0-9_]`)
illegalCharacters.ReplaceAllString(r.Org, "_")
```

WDYT?

Reply:
lgtm